nss-day-cohort-41/tabloid-fullstack-first-amendment-dancers-andy
# Search by tag pm

# Description
Search Post by tag Nav Item
## Type of change
# Testing Instructions
As a logged in user, you may click the Search By Tag nav link. All available tags will be displayed on the left of the page. Clicking a tag button will display all posts that have that tag. You may click the title of any given post to see a details page of that post. 
From any post details page, you may click the a 'tag' button and be brought directly to the search page. And a result will be displayed with all posts having that tag.
The result set will be displayed below a message that states results for {tag}. If no posts were found with that tag, the message will state that
# Checklist:
